src/FluentCheck.ts

```diff
@@ -231,3 +252,24 @@ class FluentCheckAssert<Rec extends ParentRec, ParentRec extends {}> extends Flu
       new FluentResult(false)
   }
 }
+
+class FluentCheckGenerator<Rec extends ParentRec, ParentRec extends {}> extends FluentCheck<Rec, ParentRec> {
+  constructor(
+    readonly parent: FluentCheck<ParentRec, any>,
+    config: FluentConfig,
+    readonly generator: (seed: number) => () => number,
+    readonly seed?: number
+  ) {
+    super(parent, config)
+    if (seed !== undefined)
```

Comment:
Use nullish coalescing

Reply:
Can it be used here as well? the variable being checked is the seed passed by parameter and the one being assigned is the seed present in the prng variable